src/tidalforce_odom/src/odom.py

```diff
@@ -29,6 +29,7 @@
 """
 
 import rospy
+import sys
```

Comment:
for more complex command line arguments - and automatic "help" command, check out argparse.